.nvmrc

```diff
@@ -0,0 +1 @@
+v18.17.1
```

Comment:
When I was running the project for the first time, I noticed it needed to have node v18.
This can help other users of nvm.